static/app/components/tabs/index.tsx

```diff
@@ -0,0 +1,69 @@
+import 'intersection-observer'; // polyfill
+
+import {createContext, useState} from 'react';
+import styled from '@emotion/styled';
+import {AriaTabListProps} from '@react-aria/tabs';
+import {Item} from '@react-stately/collections';
+import {TabListProps, TabListState} from '@react-stately/tabs';
+import {ItemProps, Orientation} from '@react-types/shared';
+
+import {TabList} from './tabList';
+import {TabPanels} from './tabPanels';
+
+const _Item = Item as <T>(props: ItemProps<T> & {disabled?: boolean}) => JSX.Element;
+export {_Item as Item, TabList, TabPanels};
+
+export interface TabsProps<T> extends TabListProps<T>, AriaTabListProps<T> {
```

Comment:
What exactly is the template type for here?

Reply:
It eventually [gets passed to ItemProps](https://github.com/adobe/react-spectrum/blob/e514827f49696a9c20fb819bd9610651f7017039/packages/%40react-types/shared/src/collections.d.ts#L15), meant for typing items in `react-stately`'s [dynamic collections](https://react-spectrum.adobe.com/react-stately/collections.html#dynamic-collections). We don't use dynamic collections here in Tabs but the generic template is still required.

Reply:
Can you just do this then
```suggestion
export interface TabsProps extends TabListProps<any>, AriaTabListProps<any> {
```

Reply:
I can! Just wanted to avoid using `any` in so many places.

Reply:
IMO it's okay for this

Reply:
https://github.com/getsentry/sentry/pull/38886/commits/18658e7d476137136b8bc84628b078e2f67d647e

Reply:
we're not using the type in the collection right?

Reply:
def better I think!